### DISCUSSION TITLE
lowercase email

### DISCUSSION THREAD

**Discussion by lucassebe** [faker-js/faker#1535] [Q&A] [answered]
I'm trying to set an email with lowercase like:
_*(using Cypress.io)_
 
var name = faker.name.firstName('male') + ' ' + faker.name.lastName('male');
var email = faker.internet.email(name);

but the response is always like: 
name: 'Deusimar Carvalho';
mail: 'DeusimarCarvalho45@yahoo.com'

how to set the email to lowercase only?

**Comment by Shinigami92** [comment, marked as answer]
`email.toLowerCase()`

**Reply by lucassebe** [reply]
Thanks!!
